packages/remix/test/integration/test/server/utils/helpers.ts

```diff
@@ -1,33 +1,54 @@
 import * as http from 'http';
 import { AddressInfo } from 'net';
-import { createRequestHandler } from '@remix-run/express';
-import { wrapExpressCreateRequestHandler } from '@sentry/remix';
+import { createRequestHandler as createFastifyRequestHandler } from '@mcansh/remix-fastify';
+import { createRequestHandler as createExpressRequestHandler } from '@remix-run/express';
+import { wrapExpressCreateRequestHandler, wrapFastifyCreateRequestHandler } from '@sentry/remix';
 import express from 'express';
+import fastify from 'fastify';
+import formBody from '@fastify/formbody'
+
 import { TestEnv } from '../../../../../../../dev-packages/node-integration-tests/utils';
 
 export * from '../../../../../../../dev-packages/node-integration-tests/utils';
 
-export class RemixTestEnv extends TestEnv {
-  private constructor(public readonly server: http.Server, public readonly url: string) {
-    super(server, url);
-  }
+export const enum Adapter {
+  Builtin = 'builtin',
+  Express = 'express',
+  Fastify = 'fastify',
+}
 
-  public static async init(adapter: string = 'builtin'): Promise<RemixTestEnv> {
-    const requestHandlerFactory =
-      adapter === 'express' ? wrapExpressCreateRequestHandler(createRequestHandler) : createRequestHandler;
+const adapters = {
+  [Adapter.Builtin]: createExpressRequestHandler,
```

Comment:
built-in is actually a `createRequestHandler` from `@remix-run/express`. Does this mean it actually does not need a wrapper?

Reply:
Yes, we're auto-instrumenting the built-in `createRequestHandler` from `@remix-run/node`. So this doesn't need a wrapper for built-in tests.